### ISSUE TITLE
jpegtran lossy for certain file?

### ISSUE THREAD

**Issue by JayXon** [mozilla/mozjpeg#224]
After this:
```
jpegtran -outfile 5c67195f6993c9f8d0d32d4ffe0d8e62_mozjpeg.jpg 5c67195f6993c9f8d0d32d4ffe0d8e62.jpg
```
some pixels slightly changed.

5c67195f6993c9f8d0d32d4ffe0d8e62.jpg:
![5c67195f6993c9f8d0d32d4ffe0d8e62](https://cloud.githubusercontent.com/assets/1552641/18902687/5c961d10-8509-11e6-9a38-9aa334715bcc.jpg)
5c67195f6993c9f8d0d32d4ffe0d8e62_mozjpeg.jpg:
![5c67195f6993c9f8d0d32d4ffe0d8e62_mozjpeg](https://cloud.githubusercontent.com/assets/1552641/18902690/61bc188a-8509-11e6-9ee7-91362042067f.jpg)

This file is from https://github.com/MozillaSecurity/fuzzdata/blob/master/samples/jpg/5c67195f6993c9f8d0d32d4ffe0d8e62.jpg
some other files in that directory also have this issue.

**Comment by pornel** [comment]
The input file is corrupted, so it's not entirely surprising.

**Comment by JayXon** [comment]
Yes, the file is malformed, which is good for testing edge cases.
The file can be opened by major browsers and photo viewers without error, so I'd expect mozjpeg could handle this file correctly (lossless).

**Comment by CoolOppo** [comment]
This isn't really an edge-case. It's a corrupt file. The solution here is to make mozjpeg warn when the input is corrupt, but I really don't see the point in wasting time trying to make it tiptoe around corrupt files in order to preserve the already corrupt data.
